regolith/templates/preslist.tex

```diff
@@ -0,0 +1,38 @@
+\documentclass[prl,tighten,amsmath,amssymb,floatfix]{revtex4-1}
+
+
+% Packages
+\usepackage{graphicx}
+\usepackage{times}
+\usepackage{booktabs,dcolumn}
+\usepackage{siunitx}
+
+\setlength{\parindent}{0pt}
+
+% Document
+\begin{document}
+\textbf{Presentations}
+
+\textbf{ {{-latex_safe(pi['name'][0]).upper() }}}
+\vskip
+
+\begin{enumerate}
+{% for pres in presentations %}
+pres['authors'] = ', '.join(pres['authors'])
+\item {{ latex_safe(pres['authors']) }}, {\bf {{ latex_safe(pres['title']) }} },
+{% set inst = pres['institution'] %}
+{% if pres['type'] in ['colloquium', 'seminar'] %}
+{\it {{ latex_safe(pres['type']).title() }} },
+{\it {{ latex_safe(inst['department']) }} },
```

Comment:
This asks for department but we might consider tracking department in the presentation itself, since one institution may have many departments. Ether way we need a way to specify which department is of interest.